### ISSUE TITLE
Add setting to allow item value check if weight is zero

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#444]
Nexus request from [sirlach](https://www.nexusmods.com/users/2054529)

> I use Weightless NG - https://www.nexusmods.com/skyrimspecialedition/mods/93796 - for QOL purposes. Having ore / potions / food with 0 weight is  very handy but it has 1 downside....it throws out the Value/Weight equation that I use in your mod to selectively pick up items. Would it be anywhere in the realm of possibility to add just "value" as  one of the determining factor for item pickup selection?

Response:

> maybe a way to change the treatment of weightless items to consider their value vs some threshold during the V/W checking is possible. "Weightless item must exceed value X" or something like that?

Response:

> "Weightless item must exceed value X"  sounds like just the ticket!

**Comment by SteveTownsend** [comment]
This needs MCM support but can rely on the existing V/W settings to make it available per-category - assume the weight is 1.0 and check vs the configured V/W ratio.

**Comment by SteveTownsend** [comment]
This works with the mod. I cannot edit things to be weightless easily, for some reason
```
2024-02-10 13:04:32.761    trace  75184 Must check weightless item value > 4
2024-02-10 13:04:32.761    trace  75184 TESValueForm for Spell Tome: Healing Hands/0x000a271e has 89
2024-02-10 13:04:32.761    trace  75184 V/W check for value: 89
2024-02-10 13:04:32.761    trace  75184 TESWeightForm for Spell Tome: Healing Hands/0x000a271e has weight 0 (floored at zero)
2024-02-10 13:04:32.761    trace  75184 Weightless item value too low to loot? 89
2024-02-10 13:04:32.761    trace  75184 SmartHarvest Spell Tome: Healing Hands/0x000a271e for REFR 0xff000cf3, collectible=false, type spellbook, notify true, pending 1
```
